stl/inc/flat_set

```diff
@@ -324,20 +313,15 @@ public:
         _Guard._Target = nullptr;
     }
 
-    iterator erase(iterator _Where)
-        requires (!is_same_v<iterator, const_iterator>)
-    {
-        return _Mycont.erase(_Where);
-    }
+    // NB: `erase(iterator)` is identical to `erase(const_iterator)`
     iterator erase(const_iterator _Where) {
         return _Mycont.erase(_Where);
     }
     size_type erase(const _Kty& _Val) {
         return _Erase(_Val);
     }
     template <_Different_from<_Kty> _Other>
-        requires (
-            _Keylt_transparent && !is_convertible_v<_Other, iterator> && !is_convertible_v<_Other, const_iterator>)
+        requires (_Keylt_transparent && !is_convertible_v<_Other, const_iterator>)

```

Comment:
Is it ok not to add comment for this one? Or am I adding too many repetitive comments?